docs/development/architecture.rst

```diff
@@ -8,19 +8,18 @@ Diagram
 ::
 
                                       +-----------+
-                                      | Azure     |
                                 +-----| Load Bal  |------+
                                 |     +-----------+      |
                                 |                        |
-                           +---------+              +---------+                                  
-       +-------------+     |         |              |         |    +--------------+              
-       |             |-----| Nginx   |              | Nginx   |----|              |              
-       |  File       |     +---------+              +---------+    |  File        |              
-       |  System     |          |                        |         |  System      |              
-       +-------------+     +---------+  +--------+  +---------+    +--------------+              
-              |  |         |Gunicorn |  |        |  |Gunicorn |        |   |                     
-              |  +---------|(Django) |--|Postgres|--|(Django) |--------+   |                     
-              |            +---------+  +--------+  +---------+            |                     
+                           +---------+              +---------+
+       +-------------+     |         |              |         |    +--------------+
+       |             |-----| Nginx   |              | Nginx   |----|              |
+       |  File       |     +---------+              +---------+    |  File        |
+       |  System     |          |                        |         |  System      |
```

Comment:
We could update this to say S3 / Blob Storage / Cloud Storage now, since we don't serve files from file system anymore.

Reply:
Yea, I just didn't want to redo the whole diagram :)

Reply:
We definitely need to do a nicer version of this at some point, but not today! :)